gapic/src/main/com/google/gapid/models/Resources.java

```diff
@@ -177,6 +185,17 @@ protected void onUiThread(Path.Capture result) {
     });
   }
 
+  public Service.Resource getSelectedTexture() {
+    return selectedTexture;
+  }
+
+  public void selectTexture(Service.Resource texture) {
+    if (selectedTexture != texture) {
```

Comment:
this is a pointer comparison and is unlikely to work properly for anything other than `null`s.

Reply:
Thank you, fixed